paper/src/main/java/com/devotedmc/ExilePearl/core/ExilePearlCore.java

```diff
@@ -36,21 +14,10 @@
 import com.google.common.base.Preconditions;
 import com.wimbli.WorldBorder.BorderData;
 import com.wimbli.WorldBorder.WorldBorder;
+import io.papermc.paper.plugin.configuration.PluginMeta;
```

Comment:
Curious, this interface doesn't exist in 1.18 ([reference](https://jd.papermc.io/paper/1.18/io/papermc/paper/plugin/configuration/PluginMeta.html)), but it does in 1.19 ([reference](https://jd.papermc.io/paper/1.19/io/papermc/paper/plugin/configuration/PluginMeta.html)), yet this PR is against master which is explicitly 1.18.

Reply:
Hmm I was working on 1.19 before working on this, maybe didnt cleanCache gradle before hand